pages/portfolio/[id].js

```diff
@@ -0,0 +1,53 @@
+import React from  'react'; 
+import Footer from '@/components/Footer'
+import Navbar from '@/components/Navbar';
```

Comment:
In ES6, small letters are different from capital letters. Case matters.
```suggestion
import Footer from '@/components/footer'
import Navbar from '@/components/navbar';
```